rmgpy/rmg/main.py

```diff
@@ -474,6 +479,15 @@ def initialize(self, **kwargs):
             raise ValueError('Invalid format for wall time {0}; should be DD:HH:MM:SS.'.format(self.wallTime))
         self.wallTime = int(data[-1]) + 60 * int(data[-2]) + 3600 * int(data[-3]) + 86400 * int(data[-4])
 
+        global maxProc
+        try:
+            maxProc = kwargs['maxProc']
+        except KeyError:
+            pass
+
+        if maxProc > psutil.cpu_count():
+            raise ValueError('Invalid format for user defined maximum number of procesors {0}; should be an integer and smaller or equal to your available number of cpu {1}'.format(maxProc, psutil.cpu_count()))
```

Comment:
It doesn't seem like you're checking the format here?

Reply:
Changed to 'Invalid input ...'